### PR TITLE
DLP: Added sample for de-identification with an exception list

### PR DESCRIPTION
**Pull request by dinesh-crest**
Added sample for de-identification with an exception list
Added test cases for the same

Reference: https://cloud.google.com/dlp/docs/creating-custom-infotypes-dictionary#exception_list

### CODE REVIEW

File: dlp/deIdentifyWithExceptionList.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// sample-metadata:
+//  title: Deidentify with Exception List
```
Comment (annadata) [inline dlp/deIdentifyWithExceptionList.js:18]: Please use De-identify when using in an English sentence. Here and other lines added by you.

**Review by annadata — COMMENTED** [review]
This Node.JS snippet corresponds to existing Java [snippet](https://cloud.google.com/dlp/docs/creating-custom-infotypes-dictionary#dlp_deidentify_exception_list-java).

**Review by annadata — APPROVED** [review]
LGTM.

File: dlp/deIdentifyWithExceptionList.js (hunk 1 above)
Comment (dinesh-crest) [reply dlp/deIdentifyWithExceptionList.js:18]: @annadata https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/3099 on this PR, I used the one that you mentioned, but I got a comment to change the case of the first letter. So shall i go with this one or the other one?

**Review by kweinmeister — APPROVED** [review]

**Pull request merged**